### DISCUSSION TITLE
DuckDNS: Provide IP address to run.sh

### DISCUSSION THREAD

**Discussion by hercule115** [home-assistant/frontend#8345] [Configuration]
I would like to get a way to provide the ip (v4 at least) used to update the Duck DNS server through the run.sh script.
IMHO, it is not possible today and this script is retrieving the ipv4 address by itself and update DuckDNS with this address.
In my case, this address is not correct (Not a public address) and I would like to provide the correct address to the script.
Thx in advance,

**Comment by spacegaier** [comment]
Are you talking about the HA addon or the HA integration? Initial gut feeling: Sounds more like a backend topic, as the integration should provide an option in the config flow to allow the user to specify the IP.

**Comment by frenck** [comment]
run.sh is about customizing a Docker image. You can do so, by building on top of our images, by using those as a base image.

This is not a frontend related feature request and thus locking this discussion topic.
